docker-compose.yml

```diff
@@ -122,6 +122,43 @@ services:
     labels:
       logs: "promtail"
 
+  hyperswitch-web:
+    ports:
+      - "9050:9050"
+      - "9060:9060"
+      - "5252:5252"
+    build:
+      context: ./docker
+      dockerfile: web.Dockerfile
+    networks:
+      - router_net
+    env_file:
+      - ./.env
```

Comment:
Could you provide an example of what this `.env` file would contain? Including placeholder values should be good enough.